package.json

```diff
@@ -40,6 +40,7 @@
         "eslint": "^7.18.0",
         "eslint-config-airbnb-base": "^14.2.1",
         "eslint-config-binary": "^1.0.2",
+        "eslint-config-deriv": "^1.0.0-beta.1",
```

Comment:
please install this package instead: 
https://www.npmjs.com/package/@deriv/eslint-config-deriv

this is the main repository under deriv name. 
I think after installing this in your .eslintrc file you should use the full name to extend its options like this: 
`extends: ['@deriv/eslint-config-deriv`